### ISSUE TITLE
Update GitHub repo and supporting infrastructure for HTAN2 data model

### ISSUE THREAD

**Issue by aclayton555** [ncihtan/htan-linkml#29]
Emerges from ncihtan/data-models#448 

> Question: Do we want to set up a new repo for the HTAN 2.0 data model? If so, what, if any, remaining updates should be made within the HTAN 1.0 data model repo?
> 
> Yes, new HTAN 2.0 repo and supporting infrastructure (e.g. phase 2.0 DCA) to be set up and supported in parallel with remaining maintenance of HTAN 1.0 model and infrastructure (i.e. keep the door open and working for HTAN 1.0 submissions). Stuff will be pulled over to a new repo for HTAN 2.0. Timeline TBD for final sunset/shutdown/archive of phase 1.0 infrastructure, but for now, no changes are needed to this current data model repo.

Decide to do before cloning (?) over a new repo for HTAN 2.0

-  ncihtan/htan-linkml#18 

Supporting activities

- ncihtan/htan-linkml#17

**Comment by aclayton555** [comment]
25-5: Discussed and won't do.
